apps/SageAccountWeb/src/mocks/handlers/twoFactorAuthResetHandlers.ts

```diff
@@ -8,7 +8,7 @@ const handlers = [
   http.post(
     `${getEndpoint(BackendDestinationEnum.REPO_ENDPOINT)}/auth/v1/2fa/disable`,
     () => {
-      return new HttpResponse(null, {
+      return new Response(null, {
```

Comment:
`HttpResponse` is a utility object, not a class. `new Response` is appropriate here.